.yarnrc

```diff
@@ -0,0 +1 @@
+--install.no-lockfile false
```

Comment:
looks like [`no-lockfile`](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-no-lockfile) is disabled by default and having `--install.no-lockfile false` is redundant. What was your intention of having `--install.no-lockfile false`? Can `.yarnrc` be removed if we don't need any options here?

Reply:
good catch - removed file